.github/workflows/lint.yml

```diff
@@ -0,0 +1,44 @@
+name: lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  run:
+    continue-on-error: True
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.8]
```

Comment:
we could consider to test also against other Python version, e.g. `3.9` and `3.10`, or other platforms, `windows` and `macos`.

Reply:
Ok that's a good point. I suggest we add 3.9 and 3.10 for the unit tests only and keep only 3.8 for lint and docs to keep overall CI time relatively short. For the same reason I would test on ubuntu for the moment. What do you think?

Reply:
ok